hyperkit.1

```diff
@@ -275,18 +275,18 @@ Disable MPtable generation.
 .Pp
 Firmware/bootrom configuration.
 .Bl -tag -width 10n
-.It kexec , Ns Pa kernel , Ns Oo Pa ramdisk Ns Oc , Ns Oo Pa cmdline Oc
+.It kexec , Ns Pa kernel Ns Oo , Ns Pa ramdisk Ns Oc Ns Oo , Ns Pa cmdline Oc
 Launch
 .Ar kernel
 using the Linux kexec protocol.
-.It fbsd , Ns Pa userboot , Ns Pa bootvolume , Ns Pa kernelenv
+.It fbsd , Ns Pa userboot , Ns Pa bootvolume Ns Oo , Ns Pa kernelenv Ns Oc
 Boot using the fbsd protocol.
-.It bootrom , Ns Pa path , Ns ,
+.It bootrom , Ns Pa path
 Boot using
 .Ar path
 as a firmware image. Allows booting a UEFI/tianocore binary, such as
 the one from bhyve (source: <https://github.com/freebsd/uefi-edk2.git>; binary <https://people.freebsd.org/~grehan/bhyve_uefi/>).
-.It multiboot , Ns Pa kernel , Ns Oo Pa module Ns Oo ;cmdline Ns Oc Ns Oo :module Ns Oo ;cmdline Ns Oc Ns Oc Ns ... Oc , Ns Oo Pa cmdline Ns Oc
+.It multiboot , Ns Pa kernel Ns Oo , Ns Pa module Ns Oo ;cmdline Ns Oc Ns Oo :module Ns Oo ;cmdline Ns Oc Ns Oc Ns ... Oc Ns Oo , Ns Pa cmdline Ns Oc
```

Comment:
Just below here it says:

> Note that all the commas are required in each case, even if the options are empty.

Which I think is not quite true any more. The commas are still needed for intermediate missing entries, but trailing commas are not. i.e. `kexec,kernel,,"console=foo"` still needs the `,,` for the unused initrd. Whereas `bootrom,foo` no longer needs to be `bootrom,foo,,,`.

Not sure how best to express that, but perhaps simply dropping the words `all the` to leave:

> Note that commas are required in each case, even if the options are empty.

?

> Note that commas are required even for empty options with the exception that they may be omitted for any trailing empty options.

?